client/src/components/Panels/Common/ToolSearch.vue

```diff
@@ -105,7 +105,8 @@ export default {
                 if (this.favorites.includes(q)) {
                     this.$emit("onResults", this.favoritesResults);
                 } else {
-                    const keys = ["name", "description"];
+                    // keys with sorting order
+                    const keys = { exact: 2, name: 1, description: 0 };
                     this.$emit("onResults", searchToolsByKeys(this.toolbox, keys, q));
```

Comment:
Thank you for working on this. This looks great. We should add a test for `searchToolsByKeys` and probably pre-process `this.toolbox` into a regular array `toolList` instead of rebuilding it upon search.